Nodify/Connections/BaseConnection.cs

```diff
@@ -215,16 +267,16 @@ protected override Geometry DefiningGeometry
 
         protected abstract (Point ArrowSource, Point ArrowTarget) DrawLineGeometry(StreamGeometryContext context, Point source, Point target);
 
-        protected virtual void DrawArrowGeometry(StreamGeometryContext context, Point source, Point target)
+        protected virtual void DrawArrowGeometry(StreamGeometryContext context, Point source, Point target, ConnectionDirection Direction = ConnectionDirection.Forward)
         {
-            (Point from, Point to) = GetArrowHeadPoints(source, target);
+            (Point from, Point to) = GetArrowHeadPoints(source, target, Direction);
 
             context.BeginFigure(target, true, true);
             context.LineTo(from, true, true);
             context.LineTo(to, true, true);
         }
 
-        protected virtual (Point From, Point To) GetArrowHeadPoints(Point source, Point target)
+        protected virtual (Point From, Point To) GetArrowHeadPoints(Point source, Point target, ConnectionDirection Direction)
```

Comment:
`Direction` parameter should be lowercase.

Reply:
Can I name this to `connectionDirection` to avoid conflict with `direction` that's used to assign `1d` and `-1d` in this scope?

Reply:
A better name would be arrowDirection.

Reply:
Also, I am assuming that I need to ensure that the global `Direction` `DependencyProperty` maintains its naming convention. Is it okay to enable this by adding the following line before assigning `direction`?

```
Direction = connectionDirection;
```

https://github.com/mshumayl/nodify/blob/7155f5456341f61b593ec69ed3a34da9a978c25f/Nodify/Connections/BaseConnection.cs#L284